atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/KeyValueService.java

```diff
@@ -30,7 +37,8 @@
 /**
  * A service which stores key-value pairs.
  */
-public interface KeyValueService extends Closeable {
+@Path("/keyvalue")
+public interface KeyValueService extends AutoCloseable {
```

Comment:
good question.  I think it may be jersey related somehow, I'll try to put it back and see if it breaks stuff.